src/ui/components/ObjectFrame.vue

```diff
@@ -126,6 +129,21 @@
                 return objectType.definition.cssClass;
             }
         },
+        methods: {
+            expand() {
+                let objectView = this.$refs.objectView,
+                    parentElement = objectView.$el,
+                    childElement = parentElement.children[0];
+
+                this.openmct.overlays.overlay({
+                    element: childElement,
+                    size: 'large',
+                    onDestroy() {
+                        parentElement.append(childElement);
+                    }
+                });
```

Comment:
@dtailor90 Thanks for wiring up the view large button! After clicking the expand button and closing the overlay, the object gets selected. expand() should stop propagation of the event so the object remains de-selected. Also, I would add a title "View large" for the button.